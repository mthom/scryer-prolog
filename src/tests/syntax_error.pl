:- use_module(library(debug)).
:- use_module(library(format)).

missing_dot :- write('No "." at the end of the line'), nl

:- initialization(missing_dot).
