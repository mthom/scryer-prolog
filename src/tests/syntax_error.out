   error(syntax_error(incomplete_reduction),read_term/3:6).
